src/cryptenroll/cryptenroll.c

```diff
@@ -161,6 +166,7 @@ static int parse_argv(int argc, char *argv[]) {
                 ARG_FIDO2_WITH_UP,
                 ARG_FIDO2_WITH_UV,
                 ARG_FIDO2_CRED_ALG,
+                ARG_UNLOCK_FIDO2_DEVICE,
```

Comment:
maybe use same order as in the options[] array below

Reply:
Looking at the git history of this enum, it seems that it has always been append-only regardless of the order in the options[] array? And I think too it would be best not to change the value of previously defined enum members?